Library/Formula/bazel.rb

```diff
@@ -0,0 +1,25 @@
+class Bazel < Formula
+  homepage "http://bazel.io/"
+  url "https://github.com/google/bazel.git", :tag => "0.0.2"
+  sha256 "5865356e74622c248248174acd09f38c07f3983867d7e687fa5932f2216e84c0"
+
+  head "https://github.com/google/bazel.git"
+
+  depends_on "protobuf" => :build
+  depends_on "libarchive"
+
```

Comment:
I thought of that too, however I haven't found a reliable way to depend on java yet, as java isn't available in homebrew. Adding `depends_on :java` works, however I honestly dont't know what that really does, nor was I able to find anything related to this topic in the docs or on google - maybe I'm just overlooking something?